iot_config_client/src/region_params_resolver.rs

```diff
@@ -0,0 +1,43 @@
+use crate::iot_config_client::IotConfigClientError;
+use async_trait::async_trait;
+use helium_proto::{
+    services::iot_config::RegionParamsResV1 as RegionParamsProto, BlockchainRegionParamV1, Region,
```

Comment:
```suggestion
    services::iot_config::GatewayRegionParamsResV1 as RegionParamsProto, BlockchainRegionParamV1, Region,
```

Reply:
you're right; i forgot you're working off a branch of proto here so i just saw a reference to a message that didn't exist on master and was confused how this was even compiling; carry on!